test/e2e/nix/with_bob_dependencies/bob.yaml

```diff
@@ -0,0 +1,147 @@
+variables:
```

Comment:
I like the idea of having a bobfile per test... though please strip them down to bare minimum required for the test.